### ISSUE TITLE
Force reload of stage after stage is saved depending on button clicked; edit stage page (HTML backend)

### ISSUE THREAD

**Issue by foobarlab** [foobarlab/UpStage-Video-Hack#99]
appearently the stage save buttons seem to have a bug.

new ticket derived from #97:

this is how the buttons on the stage edit page "should" work: there is one button that says "save stage", which should save the changes but not force reload; and another button that says "save and reload stage".

**Comment by helenvarley** [comment]
yes also the other buttons do not work as they should (in v2.4.2 - i'm not sure if these have already been fixed by the students in v2.4.3 but the trac ticket is still open). for example "clear stage" is supposed to clear the chat text & any drawings, but in fact what it does is unassign all the media from the stage. i will find out where the students are at with this ...

**Comment by foobarlab** [comment]
please open a new ticket for each issue and give detailed information so we do not mixup things. the purpose of each ticket is to address a specific bug or task. thanks!

**Comment by helenvarley** [comment]
issue #102 is the one about batch processing - assigning /unassigning/deleting multiple media items at once.

**Comment by foobarlab** [comment]
Issue #102 is not related to this ticket.

**Comment by helenvarley** [comment]
in beta v3, the buttons have been reworked & also the interface improved. it's still not working completely as it should - currently "save stage" and "save and reload stage" also clear the text from the chat window, which they should not do. but the "save stage" saves changes without forcing a reload, whereas "save and reload stage" asks you if you want to force a reload for anyone on the stage, and then does that.

**Comment by foobarlab** [comment]
Please do not post unrelated comments here as this is not the AUT v3-beta version. Thanks!
